frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml

```diff
@@ -1,2 +1,2 @@
-question: Will you be transparent about your financial performance, in line with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
+question: Will you be transparent about your financial performance, and comply with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
```

Comment:
It's hard to find the "government guidance". Closest I can get is this, which mentions "open book … provisions" in the final point of the first document:
https://www.gov.uk/government/publications/commercial-capability-contract-management-standards

[gov.uk/search?q=obcm](https://www.gov.uk/search?q=obcm) and [google.co.uk/search?q="obcm"+site:gov.uk](https://www.google.co.uk/search?q="obcm"+site%3Agov.uk)
don't seem to produce anything.